buns_store/src/main/java/org/example/ExampleClientController.java

```diff
@@ -0,0 +1,26 @@
+package org.example;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+@RestController
+public class ExampleClientController {
+
+    private OkHttpClient client = new OkHttpClient();
+
+    @GetMapping("/sendHello")
+    public String runHello() throws IOException {
+        Request request = new Request.Builder()
+                .url("http://localhost:8010/hello")
```

Comment:
🎉 
Fantastic work!

Reply:
As a follow up (in another PR or in here, as you decide) I would encourage you to:
- extract the host's address ` to a field whose value would be actually taken from `properties` 
- extract its port also to properties